src/libraries/Microsoft.Extensions.Options/tests/SourceGeneration.Unit.Tests/Baselines/UsingInterfaceAsPropertyTypeForLengthAttributesTests.netcore.g.cs

```diff
@@ -0,0 +1,252 @@
+
+    // <auto-generated/>
+    #nullable enable
+    #pragma warning disable CS1591 // Compensate for https://github.com/dotnet/roslyn/issues/54103
+    namespace Test
+{
```

Comment:
[nit] mis-indented

Reply:
We can fix this in another PR. I want to limit the change here as we need to port this to 8.0. Thanks for the comment.